app/api.py

```diff
@@ -1,60 +1,86 @@
-# This file contains the APIs that I created to solve the problem.
-from functools import wraps
-
-import bcrypt
-from werkzeug.security import check_password_hash
-
-from app.Model import user_data, car_data
+import datetime
+import json
 import urllib
+import bcrypt
 import requests
-import json
-from app import app, db
+from app import *
 from flask import request, jsonify
-import jwt
-import datetime
-from authentication.configuration import token_required, send_email
+from models import user, car
+from authentication.email import *
+from werkzeug.security import check_password_hash
+from flask_jwt_extended import create_access_token
+from flask_jwt_extended import create_refresh_token
+from flask_jwt_extended import get_jwt_identity
+from flask_jwt_extended import jwt_required
+from flask_jwt_extended import JWTManager
+
+jwt = JWTManager(app)
```

Comment:
can you create this globally,, as you have created DB?

Reply:
Sure, let me check it out.

Reply:
is this fixed?

Reply:
Yes, I fixed it.